readthedocs/api/v3/serializers.py

```diff
@@ -1187,8 +1192,14 @@ class Meta:
             "_links",
         )
 
+
+class OrganizationSerializerWithExpandableFields(
+    OrganizationSerializerWithoutExpandableFields
+):
+    class Meta(OrganizationSerializerWithoutExpandableFields.Meta):
         expandable_fields = {
-            "projects": (ProjectSerializer, {"many": True}),
+            # TODO: we are leaking all teams and its members to anyone who is
+            # an admin member of the organization.
             "teams": (TeamSerializer, {"many": True}),
```

Comment:
Similar here, having another endpoint will be easier `organization/{organization}/teams`, similar to `organization/{organization}/projects`.